internal/catalog_test.go

```diff
@@ -285,8 +285,8 @@ func TestUcAccMetastores(t *testing.T) {
 	assert.NotEqual(t, created.MetastoreId, currentMetastore.MetastoreId)
 
 	_, err = w.Metastores.Update(ctx, catalog.UpdateMetastore{
-		Id:   created.MetastoreId,
-		Name: RandomName("go-sdk-updated"),
```

Comment:
Highlight

Reply:
Fine. This is not necessary for Metastore (since the name is not the path parameter) but better for consistency.